From 2dfd87f2101371ff95b282cac8ede1bbca5301cc Mon Sep 17 00:00:00 2001
From: Lu Yahan <yahan@iscas.ac.cn>
Date: Wed, 27 Apr 2022 16:15:30 +0800
Subject: [PATCH] [riscv64] Remove the Dummy interface descriptor

Port commit 4ae5a813c75959fe0afe2b4ffc2b4722ad932ddf

Change-Id: I2b4fccff6bf5266cd4cf9df45cabe3a19f29ab41
Reviewed-on: https://chromium-review.googlesource.com/c/v8/v8/+/3609751
Commit-Queue: Yahan Lu <yahan@iscas.ac.cn>
Auto-Submit: Yahan Lu <yahan@iscas.ac.cn>
Reviewed-by: ji qiu <qiuji@iscas.ac.cn>
Commit-Queue: ji qiu <qiuji@iscas.ac.cn>
Cr-Commit-Position: refs/heads/main@{#80205}
---
 src/codegen/riscv64/interface-descriptors-riscv64-inl.h | 3 +--
 1 file changed, 1 insertion(+), 2 deletions(-)

diff --git a/src/codegen/riscv64/interface-descriptors-riscv64-inl.h b/src/codegen/riscv64/interface-descriptors-riscv64-inl.h
index d987269153..68f90ef0a8 100644
--- a/src/codegen/riscv64/interface-descriptors-riscv64-inl.h
+++ b/src/codegen/riscv64/interface-descriptors-riscv64-inl.h
@@ -231,8 +231,7 @@ constexpr auto ConstructStubDescriptor::registers() {
   // a3: new target
   // a1: target
   // a0: number of arguments
-  // a2: allocation site or undefined
-  return RegisterArray(a1, a3, a0, a2);
+  return RegisterArray(a1, a3, a0);
 }
 
 // static
-- 
2.35.1

